File: js/problem-1013.js
// https://www.beecrowd.com.br/judge/en/problems/view/1013

let input = require('fs').readFileSync('/dev/stdin', 'utf8'),
[a, b, c] = input.split(' ').map(value => Number(value)),
maiorAb = (a + b + Math.abs(a - b)) / 2,
max = (maiorAb + c + Math.abs(maiorAb - c)) / 2;
console.log(`${max} eh o maior`);

File: python/problem-1013.py
# https://www.beecrowd.com.br/judge/en/problems/view/1013

a, b, c = map(lambda inp: int(inp), input().split())
maiorAB = (a + b + abs(a - b)) // 2
maior = (maiorAB + c + abs(maiorAB - c)) // 2

print(f"{maior} eh o maior")